keyboards/planck/keymaps/kylejm/keymap.c

```diff
@@ -0,0 +1,149 @@
+ * |      |   1  |   2  |   3  |   4  |   5  |      |  ⏯  |  ⏮️   |  ⏭️  |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |   6  |   7  |   8  |   9  |   0  |      | Left | Right|  Up  | Down |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      | Mute | Vol- | Vol+ |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT_planck_grid(
+    _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_MPLY, KC_MPRV, KC_MNXT, _______, _______,
+    _______, KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______, KC_LEFT, KC_RGHT, KC_UP,   KC_DOWN, _______,
+    _______, _______, _______, _______, _______, _______, _______, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |Reset |      |      |      |      |      |      |      |      |      |      |Debug |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_planck_grid(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,
+    RESET  , _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, DEBUG  ,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______
+)
+
+};
+
+// midi mode looks cool
+// what do the function keys do on macOS? F11 is expose
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case UNDO:
+      if (record->event.pressed) {
+        SEND_STRING(SS_LGUI("z"));
+        return false;
+      }
+      break;
```

Comment:
This can be replaced with `LGUI(KC_Z)` in the keymap directly.

Reply:
👍 done